src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_SORT_ACKNOWEDGEMENT = "The have been sorted.";
+
+    public CommandResult execute() {
+        this.addressBook.sort();
+        //List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.